src/nixpacks/app.rs

```diff
@@ -171,10 +171,7 @@ impl App {
 
     /// Convert an absolute path to a path relative to the app source directory
     pub fn strip_source_path(&self, abs_path: &Path) -> Result<PathBuf> {
-        let source_str = match self.source.to_str() {
-            Some(s) => s,
-            None => bail!("Failed to parse source path"),
-        };
+        let source_str = self.source.to_str().unwrap();
```

Comment:
Why change it from what it was before? Which was a `bail!` with error message

Reply:
Unwrapping will also panic as opposed to returning a `Result`

Reply:
Resolved